mail/api.py

```diff
@@ -29,12 +38,11 @@ def _mailgun_request(cls, request_func, endpoint, params):
             requests.Response: HTTP response
         """
         mailgun_url = '{}/{}'.format(settings.MAILGUN_URL, endpoint)
-        emails_params = {'from': settings.MAILGUN_FROM_EMAIL}
-        emails_params.update(**params)
+        params.update(cls.get_base_params())
         return request_func(
             mailgun_url,
-            auth=('api', settings.MAILGUN_KEY),
-            data=dict(**emails_params)
+            auth=cls._basic_auth_credentials,
+            data=dict(**params)
```

Comment:
This was how it was before I edited it - I'll leave it in place in case it was done this way for a reason.